problem5.py

```diff
@@ -0,0 +1,32 @@
+"""
+Created on Sun Mar 18 10:34:22 2018
+@author: patricio
+
+is_divisible_by: given a number and a list of numbers, it returns true if the
+number is divisible by all the numbers in the list or false if not.
+"""
+import time
+
+def is_divisible_by(number, list_of_numbers):
```

Comment:
This is kind of inefficient. Try using another method that utilizes prime numbers.